docs/reference/settings/ml-settings.asciidoc

```diff
@@ -100,6 +100,13 @@ dynamic setting in version 7.1. As a result, changes to its value after node
 startup are used only after every node in the cluster is running version 7.1 or
 higher. The maximum permitted value is `512`.
 
+`xpack.ml.nightly_maintenance_requests_per_second`::
+(<<cluster-update-settings,Dynamic>>) The rate at which the nightly maintenance task
+deletes expired model snapshots and results. The setting is a proxy to the
+[requests_per_second](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html#_throttling_delete_requests)
+parameter used in the Delete by query requests and controls throttling.
+Valid values must be greater than `0.0` or equal to `-1.0`. Defaults to `-1.0`
```

Comment:
If I understand correctly, `-1.0` means that the setting is disabled. If it's correct then I would add it to the docs.

```suggestion
Valid values must be greater than `0.0` or equal to `-1.0`. Defaults to `-1.0`, which means the setting is disabled.
```

Reply:
The setting is not exactly the same as the delete by query equivalent. For delete by query `-1.0` means the setting is disabled. In this case `-1.0` means ml performs a calculation and uses that value for `requests_per_second`. It is never completely disabled.

I changed it to 
```
Valid values must be greater than `0.0` or equal to `-1.0` where `-1.0` means a default value
is used. Defaults to `-1.0`
```